lib/encryptor.js

```diff
@@ -0,0 +1,19 @@
+var cipher = require('./cipher.js');
+
+exports.encryptMessageContent = encryptMessageContent;
```

Comment:
why do we need this in separate file as it is just a wrapper around  lib/cipher.js functions?